support-frontend/assets/components/orderSummary/contributionsOrderSummaryContainer.tsx

```diff
@@ -1,33 +1,50 @@
 import { checkListData } from 'components/checkoutBenefits/checkoutBenefitsListData';
-import {
-	type ContributionType,
-	getAmount,
-	type RegularContributionType,
-} from 'helpers/contributions';
+import type { RegularContributionType } from 'helpers/contributions';
+import { type ContributionType, getAmount } from 'helpers/contributions';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import { currencies } from 'helpers/internationalisation/currency';
+import type { Promotion } from 'helpers/productPrice/promotions';
 import { isSupporterPlusFromState } from 'helpers/redux/checkout/product/selectors/isSupporterPlus';
 import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
 import { getUserSelectedAmount } from 'helpers/redux/checkout/product/selectors/selectedAmount';
 import { useContributionsSelector } from 'helpers/redux/storeHooks';
 import { getLowerBenefitsThreshold } from 'helpers/supporterPlus/benefitsThreshold';
 import { trackComponentClick } from 'helpers/tracking/behaviour';
+import { supporterPlusLegal } from 'pages/supporter-plus-landing/setup/supporterPlusLegal';
 import type { ContributionsOrderSummaryProps } from './contributionsOrderSummary';
 
 type ContributionsOrderSummaryContainerProps = {
 	inThreeTier: boolean;
 	renderOrderSummary: (props: ContributionsOrderSummaryProps) => JSX.Element;
+	promotion?: Promotion;
 };
 
 function getTermsConditions(
+	countryGroupId: CountryGroupId,
 	contributionType: ContributionType,
 	isSupporterPlus: boolean,
+	promotion?: Promotion,
 ) {
 	if (contributionType === 'ONE_OFF') return;
 	const period = contributionType === 'MONTHLY' ? 'month' : 'year';
 
 	if (isSupporterPlus) {
 		return (
 			<>
+				{promotion && (
+					<p>
+						You’ll pay{' '}
+						{supporterPlusLegal(
+							countryGroupId,
+							contributionType,
+							'/',
```

Comment:
What's the logic of where we use ` per ` vs `/`?

Reply:
T&C's want `per`, `/ `used in summaryContainer description, if we expanded to use on Landing or than kyou could also have these differences

Reply:
Ah - OK. I'm still not sure I get _**why**_ we use one or the other, but all good.